tests/tests_wikipedia_article_page.py

```diff
@@ -0,0 +1,88 @@
+import time
+import allure
+from selenium.webdriver import Keys
+
+
+@allure.title("Verify that hide button is hiding side menu")
+def test_wikipedia_article_page_hide_button(pages):
+    with allure.step("insert text in search box"):
+        time.sleep(1)
```

Comment:
Please, try to never use sleep() for waiting. Could you please use WebDriverWait instead of it? Or retry function if it will not work?